ui/src/app/edge/history/consumption/widget.component.ts

```diff
@@ -78,30 +78,34 @@ export class ConsumptionComponent extends AbstractHistoryWidget implements OnIni
                     !component.isEnabled == false);
             for (let component of this.evcsComponents) {
                 channels.push(
-                    new ChannelAddress(component.id, 'ActiveConsumptionEnergy'),
+                    new ChannelAddress(component.id, this.CHANNEL_ID_ACTIVE_CONSUMPTION_ENERGY),
                 )
             }
 
             this.consumptionMeterComponents = config.getComponentsImplementingNature("io.openems.edge.meter.api.SymmetricMeter")
                 .filter(component => component.isEnabled && config.isTypeConsumptionMetered(component));
             for (let component of this.consumptionMeterComponents) {
                 channels.push(
-                    new ChannelAddress(component.id, 'ActiveProductionEnergy'),
+                    new ChannelAddress(component.id, this.CHANNEL_ID_ACTIVE_CONSUMPTION_ENERGY),
                 )
             }
             resolve(channels);
         });
     }
 
+    public getChannelAddressString(component: EdgeConfig.Component, channelId: string): string {
```

Comment:
hey, i fell over your PR: template called functions and variables can be protected